### DISCUSSION TITLE
Butler Authentication to qliksense using JWT

### DISCUSSION THREAD

**Discussion by wieeem** [ptarmiganlabs/butler-sos#520] [Q&A] [answered]
Hello,
I would like to know if Butler supports JWT authentication to Qliksense server. 
If JWT authentication is available, I would appreciate any guidance or documentation on how to configure Butler to use JWT for authentication with Qlik Sense.

Thank you for your time and assistance

**Comment by mountaindude** [comment, marked as answer]
Hi there,

Butler does currently not support JWT authentication with Qlik Sense.
It's a very good idea though and on the long-term todo list, no concrete time frame though.

If you *are* working with JWTs and Qlik Sense, you may want to take a look at qs-jwt too,  https://github.com/ptarmiganlabs/qs-jwt
It is an open source tool for creating JWTs that can be used with client-managed Qlik Sense as well as Qlik Cloud.
